Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView.swift

```diff
@@ -0,0 +1,165 @@
+            moreButton.setTitle(moreButtonTitle, for: .normal)
+        }
+    }
+
+    public var okayButtonTitle: String = "" {
+        didSet {
+            okayButton.setTitle(okayButtonTitle, for: .normal)
+        }
+    }
+
+    // MARK: - Setup
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+}
+
+// MARK: - Private
+
+private extension ConsentTransparencyView {
+    func setup() {
+        addSubview(scrollView)
+        scrollView.addSubview(contentView)
+
+        contentView.addSubview(imageView)
+        contentView.addSubview(headerLabel)
+        contentView.addSubview(detailLabel)
+        addSubview(buttonStackView)
+
+        NSLayoutConstraint.activate([
+            scrollView.topAnchor.constraint(equalTo: topAnchor),
+            scrollView.bottomAnchor.constraint(equalTo: buttonStackView.topAnchor, constant: -.mediumLargeSpacing),
+            scrollView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: trailingAnchor),
+
+            contentView.topAnchor.constraint(equalTo: scrollView.topAnchor),
+            contentView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor),
+            contentView.leadingAnchor.constraint(equalTo: scrollView.leadingAnchor),
+            contentView.trailingAnchor.constraint(equalTo: scrollView.trailingAnchor),
+            contentView.widthAnchor.constraint(equalTo: scrollView.widthAnchor),
+
+            imageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: .mediumLargeSpacing),
+            imageView.leadingAnchor.constraint(greaterThanOrEqualTo: contentView.leadingAnchor, constant: .mediumLargeSpacing),
+            imageView.trailingAnchor.constraint(lessThanOrEqualTo: contentView.trailingAnchor, constant: -.mediumLargeSpacing),
+            imageView.centerXAnchor.constraint(equalTo: contentView.centerXAnchor),
+
+            headerLabel.topAnchor.constraint(equalTo: imageView.bottomAnchor, constant: .mediumLargeSpacing),
+            headerLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: .mediumLargeSpacing),
+            headerLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -.mediumLargeSpacing),
+
+            detailLabel.topAnchor.constraint(equalTo: headerLabel.bottomAnchor, constant: .mediumSpacing),
+            detailLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: .mediumLargeSpacing),
+            detailLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -.mediumLargeSpacing),
+            detailLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -.smallSpacing),
+
+//            buttonStackView.topAnchor.constraint(equalTo: detailLabel.bottomAnchor, constant: .largeSpacing),
```

Comment:
Remove this commented line if we are not using it?

Reply:
Fixed! ☑️